src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -445,6 +446,52 @@ Copyright (c) .NET Foundation. All rights reserved.
     </Csc>
   </Target>
 
+  <Target Name="_GetAssemblyInfoFromTemplateFile">
+
+    <PropertyGroup>
+      <_CoreGenerateSatelliteAssembliesTemplateFile>$(IntermediateOutputPath)$(TargetName)$(TargetExt)</_CoreGenerateSatelliteAssembliesTemplateFile>
```

Comment:
It's actually not necessarily relative, just possibly relative. IntermediateOutputPath is often setup to be a full path.